contracts/contracts/rem.system/src/producer_pay.cpp

```diff
@@ -125,7 +133,8 @@ namespace eosiosystem {
             _producers.modify(prod, same_payer, [&](auto& p) {
                p.last_expected_produced_blocks_update = timestamp;
             });
-            _gstate.last_schedule[i] = std::make_pair(prod_name, prod.total_votes / _gstate.total_active_producer_vote_weight);
+            const double share = prod.total_votes / _gstate.total_active_producer_vote_weight;
+            _gstate.last_schedule[i] = std::make_pair(prod_name, std::floor(share * 100000.0) / 100000.0);
```

Comment:
@me-vadim could you please leave a comment why do we need such rounding

Reply:
Done